llvm/include/llvm/SYCLLowerIR/ComputeModuleProperties.h

```diff
@@ -0,0 +1,45 @@
+//===- ComputeModuleProperties.h - compute properties for module ----------===//
```

Comment:
This file computes both module properties and symbols. May be the filename and description needs to include symbols as well? Another option will be to split and add a second file - ComputeModuleSymbols.cpp. 

Thanks

Reply:
both functions are basically guaranteed to always be used together so i'm not sure if splitting it into two files makes sense, and i couldn't think of a more descriptive name for a single file that isn't super long and over the character limit, so i'm planning on leaving this for now even if it isn't ideal. let me know if you prefer i address this now.

Reply:
+1 for renaming if it's not much of a hassle. Will help someone new to quickly understand what we are doing. Thanks

Reply:
renamed in latest commit, thanks